pkg/storage/kubernetes/client.go

```diff
@@ -212,7 +212,7 @@ func validateSubscriberVersion(apiVersion string, sub subscriber.Subscriber) boo
 	}
 
 	for _, v := range sub.SubStore.Store {
-		if !isVersionsCompatible(v.GetVersion(), apiVersion) {
+		if !isVersionsCompatible("2.0", apiVersion) {
```

Comment:
function validateSubscriberVersion() and isVersionsCompatible() should be removed.

Reply:
apiVersion parameter should be removed from CreateConfigMap()